config/nim.cfg

```diff
@@ -16,12 +17,21 @@ cc = gcc
 hint:LineTooLong:off
 #hint:XDeclaredButNotUsed:off
 
-# Examples of how to setup a cross-compiler:
```

Comment:
`cross-compiler` still needs to be mentioned in comments for searchability

Reply:
Done